web/frontend/src/components/slam.vue

```diff
@@ -117,18 +118,10 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
 };
 
 const executeMoveOnMap = async () => {
-  if (updatedDest === false) {
```

Comment:
I assume this check is no longer necessary because of the updateDest check in moveClick

Reply:
yep